app/packages/core/src/components/Modal/TooltipInfo.tsx

```diff
@@ -121,7 +120,7 @@ export const TooltipInfo = React.memo(() => {
           {detail.label.tags && detail.label.tags.length > 0 && (
             <TagInfo key={"tags"} tags={detail.label?.tags} />
           )}
-          <Component key={"attrs"} detail={detail} />
+          <AllOverlayInfo key={"attrs"} detail={detail} />
```

Comment:
`Component` would resolve to like `HeatmapInfo` which would only show details of one layer.